### PR TITLE
#1 Adding initial Database, Table, and Data. 

### PR DESCRIPTION
**Pull request by Amtresu**
## In this project I've practiced my SQL knowledge. 

- Used GitFlow. ✅ 
- Documented work in a professional way. ✅ 
- Used template to generate repo. ✅ 
- Created a database called `vet_clinic`. ✅ 
- Created a table called `animals `with the correct columns.✅  
- Copied my SQL from pgadmin4 into my repo. ✅ 
- Inserted Data. ✅
- Wrote and tested Queries.✅  
image
image
image
image
image
image
image
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+SELECT * from animals WHERE TO_CHAR(date_of_birth, 'yyyy') BETWEEN '2016' and '2019';
+SELECT name from animals where neutered = false and escape_attempts > 3;
```
Comment (Bondok6) [inline queries.sql]: - [x] According to the requirements, we need to select name of all animals that are neutered `neutered = true` and have less than 3 escape attempts. `escape_attempts < 3`, kindly update it 💯

image

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+SELECT date_of_birth from animals where name IN ('Agumon', 'Pikachu'); 
+SELECT name, escape_attempts from animals where weight_kg > 10.5;
+SELECT name from animals where neutered = true;
+SELECT * from animals WHERE name LIKE '%Gabumon';
```
Comment (Bondok6) [inline queries.sql]: - [x] According to the requirements, we need to find all animals not named Gabumon. So I suggest using `not LIKE` or `!=`  or  `<>`💯

image

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @Amtresu 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Using GitHub flow ✅
- Your project is professional ✅

## Required Changes ♻️
_Check the comments under the review._ ⬇


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql [hunk 3]
```diff
@@ -2,9 +2,9 @@
 
 SELECT * from animals WHERE name LIKE '%mon';
 SELECT * from animals WHERE TO_CHAR(date_of_birth, 'yyyy') BETWEEN '2016' and '2019';
-SELECT name from animals where neutered = false and escape_attempts > 3;
+SELECT name from animals where neutered = false and escape_attempts < 3;
```
Comment (Bondok6) [inline queries.sql:5]: - [OPTIONAL] I appreciate that you implemented the changes 💯, However,  the neutered here should be true `neutered = true`. But for the hard work you've done, your project deserves to be approved, and I am sure that you will fix this minor issue. 👍

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @Amtresu 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 3 above)
Comment (Amtresu) [reply queries.sql:5]: Thanks for pointing that out again. Cheers

**Pull request merged**